source/applications/dependency-injection.md

```diff
@@ -191,11 +191,17 @@ export default Ember.Component.extend({
 
 ## Factory Instance Lookups
 
-The vast majority of Ember registrations and lookups are performed implicitly.
+You rarely need to write code to look up objects in an Ember application because
```

Comment:
I removed the concept of "you rarely need to do this" and "in some cases you might do this" for now. I'd rather just tell people how to do it and not talk about how often they should look to this technique unless we have some clear guidelines.

Reading again I feel that "look up objects" is less insider than the other phrases I'm reading for this in the docs. However, I do want to be as consistent as possible with the rest of the documentation so I've changed that. I went with @mixonic's "fetching an instantiated factory".